ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -300,7 +300,8 @@ bool CurlNetworking::sendRequest(shared_ptr<CurlRequest> curlRequest, folly::dyn
     curl_easy_setopt(curl, CURLOPT_PROGRESSDATA, curlRequest.get() );
     curl_easy_setopt(curl, CURLOPT_PROGRESSFUNCTION, progressCallbackCurlWrapper);
   }
-  setHeaders(curlRequest, headers);
+  if(headers != nullptr)
+    setHeaders(curlRequest, headers);
```

Comment:
setHeaders is a public API of the sdk module, which means any client can call this API to set headers . so should the check be inside setHeaders function?